src/util.rs

```diff
@@ -138,68 +133,6 @@ fn str_from_mac_addr() {
                "12:34:56:78:09:ab");
 }
 
-#[cfg(target_os = "linux")]
-fn sockaddr_to_network_addr(sa: *const sockets::SockAddr) -> (Option<MacAddr>, Option<IpAddr>) {
```

Comment:
They have been moved to unix_interfaces.rs but for some reason they were left here as well.